drivers/soundwire/stream.c

```diff
@@ -1859,3 +1862,102 @@ int sdw_deprepare_stream(struct sdw_stream_runtime *stream)
 	return ret;
 }
 EXPORT_SYMBOL(sdw_deprepare_stream);
+
+static int set_stream(struct snd_pcm_substream *substream,
```

Comment:
`sdw_set_stream`?

Reply:
static -> no prefix
And sdw_set_stream is already used for something else